onnxruntime/contrib_ops/cuda/decoder/decoder_masked_multihead_attention.cc

```diff
@@ -139,6 +139,7 @@ Status DecoderMaskedMultiHeadAttention<T1, T2>::ComputeInternal(OpKernelContext*
   } else {
     // Sanity check
     ORT_ENFORCE(past_present_share_buffer_);
+    ORT_ENFORCE(past_key != nullptr && past_value != nullptr);
```

Comment:
I think past_key is used earlier as well. We probably should put this check before line 111.

Reply:
use self-defined shape instead of past_key/value's at line 111